idunn/utils/categories.yml

```diff
@@ -1,144 +1,602 @@
+        raw_filters:
+          - "bicycle_rental,*"
+        pj_what: "location de vélo"
+
+    museum:
+        regex: "^(museum|musee)"
+        raw_filters:
+          - "museum,*"
+        pj_what: "musée"
         pj_filters:
-            - "infrastructures de sports et loisirs"
-            - "piscines (établissements)"
-            - "clubs de sport"
+          - "musées"
+
+    shop_jewelry:
         raw_filters:
-            - "sports_centre,*"
-        regex: "fitness|muscu|piscine"
-    station:
+          - "*,jewelry"
+        pj_what: "bijouterie"
+
+    shop_newsagent:
         raw_filters:
-            - "railway,station"
-            - "railway,halt"
-        regex: "^gare( |$)"
-    recycling:
+          - "*,newsagent"
+          - "*,kiosk"
+        pj_what: "kiosque à journaux"
+
+    swimming:
         raw_filters:
-            - "recycling,*"
-        regex: "recycl|tri selectif|dechett?err?ie"
-        match_brand: True
-    parking:
+          - "swimming,*"
+        pj_what: "piscine"
+
+    shop_furniture:
         raw_filters:
-            - "parking,*"
-        regex: "parking"
-    place_of_worship:
+          - "*,furniture"
+          - "*,bed"
+        pj_what: "magasin de meubles"
+
+    shop_books:
         raw_filters:
-            - "place_of_worship,*"
-        regex: "cathedral|church|eglise|mosque|synagogue|temple"
+          - "*,books"
+        pj_what: "librairie"
+
+    shop_laundry:
+        raw_filters:
+          - "*,laundry"
+        pj_what: "blanchisserie"
+
+    shop_sports:
+        raw_filters:
+          - "shop,sports"
+          - "shop,bicycle"
+        pj_what: "magasin de sport"
+
+    theatre:
+        regex: "^theat"
+        raw_filters:
+          - "theatre,*"
+        pj_what: "théâtre"
+        pj_filters:
+          - "salles de concerts, de spectacles"
+
+    veterinary:
+        raw_filters:
+          - "veterinary,*"
+        pj_what: "vétérinaire"
+
+    shop_greengrocer:
+        raw_filters:
+          - "*,greengrocer"
+        pj_what: "marchant de fruits et légumes"
+
+    shop_garden_centre:
+        raw_filters:
+          - "*,garden_centre"
+        pj_what: "magasin de jardinage"
+
+    arts_centre:
+        raw_filters:
+          - "arts_centre,*"
+          - "gallery,*"
+        pj_what: "galerie d'art"
+
+    shop_electronics:
+        raw_filters:
+          - "*,electronics"
+          - "*,computer"
+        pj_what: "magasin d'électronique"
+
+    cinema:
+        regex: "^cine"
+        raw_filters:
+          - "cinema,*"
+        pj_what: "cinéma"
+        pj_filters:
+          - "salles de cinéma"
+
+    university:
+        regex: "^universit|educat"
+        raw_filters:
+          - "college,*"
+        pj_what: "université"
+
+    shop_travel_agency:
+        raw_filters:
+          - "*,travel_agency"
+        pj_what: "agence de voyage"
+
+    shop_other:
+        raw_filters:
+          - "shop,*"
+        pj_what: "magasin"
```

Comment:
Does that really make sense to provide a list of results for such a wide category ?